test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -300,6 +302,173 @@ void testLiteInterpreterBuiltinFunction() {
   AT_ASSERT(str == expected);
 }
 
+void testLiteInterpreterModuleInfoBasic() {
+  Module m("M");
+  m.define(R"JIT(
+    def forward(self, x):
+      return 2 * x
+  )JIT");
+
+  std::stringstream ss;
+  m._save_for_mobile(ss, {}, true);
+  mobile::Module bc = _load_for_mobile(ss);
+
+  std::unordered_set<std::string> module_debug_info_set;
+  size_t pc = 0;
+  while (true) {
+    std::string module_info = bc.get_forward_method_debug_info(pc);
+    if (module_info == "<no module info for pc>") {
```

Comment:
I think we can do testing as I described below where instead of using pc to extract this we can just ask do `bc.get_current_pc_debug_info()`. Check my comment later.

Reply:
I think for now you can just to `try.. catch` clause and populate it that way. We will have to fix the interface more appropriately later.